report/v0.2.0/symflower_symbolic-execution@golang@@golang-plain

```diff
@@ -0,0 +1,31 @@
+2024/04/03 23:07:14 Evaluating model "symflower/symbolic-execution" using language "golang" and repository "/home/zimmski/symflower/src/github.com/symflower/eval-symflower-codegen-testing/testdata/golang/plain"
+2024/04/03 23:07:14 $ symflower unit-tests --workspace /tmp/eval-codegen-testing894896298/plain plain.go
+Checking for updates
+There is a new version of symflower available! Please run `symflower update`.
+Analyzing workspace /tmp/eval-codegen-testing894896298/plain/
+Search for Go files
+Load dependency stdlib:@dev
+Found 1 Go files
+Found 0 problems in Go files
+plain.go: found 1 symbols
+plain.go: plain.plain: computing test cases
+plain.go: plain.plain: computed 1 unit tests
+plain.go: plain.plain: found 0 problems
+plain.go: generated unit test file plain_symflower_test.go
+Analyzed 1 out of 1 source files
+Had 0 errors that block a full analysis
+Generated 1 test
+Found 0 potential problems
+Give us your feedback and let us know how we can improve Symflower at hello@symflower.com or https://github.com/symflower/symflower. Thanks so much for your help!
+2024/04/03 23:07:14 $ gotestsum --format standard-verbose --hide-summary skipped -- -cover -v -vet=off ./...
+=== RUN   TestSymflowerPlain
+=== RUN   TestSymflowerPlain/#00
+--- PASS: TestSymflowerPlain (0.00s)
+    --- PASS: TestSymflowerPlain/#00 (0.00s)
+PASS
+coverage: 100.0% of statements
+ok      plain   0.001s  coverage: 100.0% of statements
+
+DONE 2 tests in 0.151s
+2024/04/03 23:07:15 Evaluated model "symflower/symbolic-execution" using language "golang" and repository "/home/zimmski/symflower/src/github.com/symflower/eval-symflower-codegen-testing/testdata/golang/plain": encountered 0 problems
+2024/04/03 23:07:15 Evaluating models and languages
```

Comment:
That line needs to go in the evaluation.log

Reply:
Also, missing the evaluation.log which should contain all remaining logs (so minus the logs of the models)